src/api/tests/createCustomer.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { test } from "fixtures/api-services.fixture";
+import { STATUS_CODES } from "data/statusCodes";
+import { generateCustomerData } from "data/customers/generateCustomer.data";
+import { TAGS } from "data/testTags.data";
+import { validateResponse } from "utils/validations/responseValidation";
+
+import { positiveTestCases, negativeTestCases } from "data/customers/customerTestCases.data";
+
+test.describe("[API] [Customers] Positive Tests", () => {
+
+    let token = "";
+    let id = "";
+
+    test.beforeEach(async ({ signInApiService }) => {
+        token = await signInApiService.loginAsLocalUser();
+    });
+
+    test.afterEach(async ({ customersApiService }) => {
+        await customersApiService.deleteCustomer(id, token);
+    });
+
+    positiveTestCases.forEach(({ name, data }) => {
+        test(`Should create customer: ${name}`,
+            { tag: [TAGS.API, TAGS.PRODUCTS, TAGS.SMOKE, TAGS.REGRESSION] },
+            async ({ customersController }) => {
+
+                const response = await customersController.create(data, token);
+                validateResponse(response, STATUS_CODES.CREATED, true, null);
+
+                id = response.body.Customer._id;
+            })
+    });
+});
+
+test.describe("[API] [Customers] Negative Tests", () => {
```

Comment:
Тут тоже название метода нужно) например

```suggestion
test.describe("[API] [Customers] [Create] Negative Tests", () => {
```